commit 2cd9cb9e18671cff4c3ec22d0566d452b01edf79
Author: Vincent Batts <vbatts@hashbangbash.com>
Date:   Fri Aug 4 09:39:33 2017 -0400

    build_recipe: carry lightstep patch
    
    this is really only for building a distribution package with newer
    gcc-c++, like 7.1.1.
    
    Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

diff --git a/ci/build_container/build_recipes/lightstep.sh b/ci/build_container/build_recipes/lightstep.sh
index 5ace138..08b6399 100755
--- a/ci/build_container/build_recipes/lightstep.sh
+++ b/ci/build_container/build_recipes/lightstep.sh
@@ -7,6 +7,9 @@ VERSION=0.36
 wget -O lightstep-tracer-cpp-$VERSION.tar.gz https://github.com/lightstep/lightstep-tracer-cpp/releases/download/v${VERSION//./_}/lightstep-tracer-cpp-$VERSION.tar.gz
 tar xf lightstep-tracer-cpp-$VERSION.tar.gz
 cd lightstep-tracer-cpp-$VERSION
+# see https://github.com/lyft/envoy/issues/1387 for progress
+wget -O ../lightstep-missing-header.diff https://raw.githubusercontent.com/vbatts/copr-build-envoy/2eea2906bfc51a858cf55b894f0ba45f958058e3/lightstep-missing-header.diff
+patch -p0 < ../lightstep-missing-header.diff
 # Added for legacy compatibility, should not be needed in new build recipes.
 [ -z "$PROTOBUF_BUILD" ] && PROTOBUF_BUILD="$THIRDPARTY_BUILD"
 ./configure --disable-grpc --prefix=$THIRDPARTY_BUILD --enable-shared=no \
